parser/parser.y

```diff
@@ -51,10 +51,11 @@ import (
 %token	<ident>
 
 	/*yy:token "%c"     */
-	identifier  "identifier"
-	asof        "AS OF"
-	toTimestamp "TO TIMESTAMP"
-	memberof    "MEMBER OF"
+	identifier           "identifier"
+	asof                 "AS OF"
+	toTimestamp          "TO TIMESTAMP"
+	memberof             "MEMBER OF"
+	optionallyEnclosedBy "OPTIONALLY ENCLOSED BY"
```

Comment:
not sure we can't match `OPTIONALLY        ENCLOSED        BY` (more space between words) now

Reply:
we can and should, see lexer.go and TestLoadDataRestore